library/pdf_creator.class.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * PdfCreator class
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
+use Knp\Snappy\Pdf;
+use Symfony\Component\Debug\ExceptionHandler;
+
+class PdfCreator
+{
+
+    private $binaryPath;
+
+    private $tempPath;
+
+    private function getBinaryPath()
+    {
+        global $webserver_root;
+        // This will not necessarily reflect actual machine bus width but php bus size.
+        $intsize = strlen(decbin(~ 0));
+        if (empty(strstr(php_uname("m"), '64'))) {
+            $bit = "32bit";
+        } else {
+            $bit = "64bit";
+        }
+        try {
```

Comment:
Prob makes more sense to just bring in all the wkhtmltopdf binaries via composer(looks like that is what is suggested on their repo). Note that even if you create it as a submodule, all the binaries will still end up in the package release, which is why I am a bit unclear to the benefits of this. Whereas composer can be more flexible in this regard(that being said, would still plop all the binaries via composer into the vendor path).